README.md

```diff
@@ -23,12 +23,13 @@ It is also easy to graph data in this format. The following example is a histogr
 1. `nvm use` (if you use NVM)
 2. `npm install`
```

Comment:
When I run this git is in a dirty state because the version changes in package-lock.json. I suspect we should .gitignore it...

Reply:
I updated the package-lock ✅. It's not meant to be gitignored, it exists so that people get the same versions of packages on different computers